packages/eslint-config-custom/index.js

```diff
@@ -59,6 +59,7 @@ module.exports = {
         ],
       },
     ],
+    '@typescript-eslint/restrict-template-expressions': ['error', { allowNumber: true }],
```

Comment:
New eslint rule: https://typescript-eslint.io/rules/restrict-template-expressions/

I think this makes sense for everything except for numbers. I think we all know that putting a number into a string template stringifies it and there isn't any crazy things like `[object Object]` to worry about.